lizardbot/screens.py

```diff
@@ -55,11 +55,54 @@ async def add_default_keyboard(
         return keyboard
 
 
+class GetFio(BaseScreen, RouteMixin):
+    """Экран для ввода ФИО преподавателя."""
+
+    routes: tuple[tuple[set[State], State]] = (
```

Comment:
```suggestion
    routes: 'Routes' = (
```
Если хотите добавлять типизацию, то делайте это в отдельном коммите